### PR TITLE
Diffmerge fallback

### PR DESCRIPTION
**Pull request by natalia-zelazna**
Hi, I added a service executed just before reboot/halt/shutdown, even after the final target. I have a concern regarding bumping dattobd version, thus I commented it out in spec for the time being.

### CODE REVIEW

File: dist/dattobd.spec [hunk 1]
```diff
@@ -572,7 +597,14 @@ rm -rf %{buildroot}
 %endif
 %endif
 
+
+
+
+
 %changelog
+# * Tue Oct 31 2023 Natalia Zelazna <natalia.zelazna@datto.com> - 0.11.4
```
Comment (Swistusmen) [inline dist/dattobd.spec:605]: I would leave version as it is (0.11.3)

File: dist/dattobd.spec [hunk 2]
```diff
@@ -7,7 +7,15 @@
 %global _kmod_src_root %{_usrsrc}/%{name}-%{version}
 
 # Location for systemd shutdown script
-%global _systemd_shutdown /lib/systemd/system-shutdown
+# "%{_vendor}" == "redhat" covers rhel, centos and fedora 
+# Ubuntu18 doesn't have /usr/lib/systemd/, but Ubuntu20 has both locations with the same content
```
Comment (Swistusmen) [inline dist/dattobd.spec:11]: I don't get it, from code it looks like /usr/lib/systemd/ is present for redhat, but in comments is that it is present for ubuntu20

File: dist/dattobd.spec [hunk 3]
```diff
@@ -489,6 +509,11 @@ rm -rf %{buildroot}
 
 # Install systemd shutdown script
 %{_systemd_shutdown}/umount_rootfs.shutdown
+%{_systemd_services}/umount-rootfs.service
+
+
+%post 
+ln -s %{_systemd_services}/umount-rootfs.service   %{_systemd_services}/reboot.target.wants/umount-rootfs.service
```
Comment (Swistusmen) [inline dist/dattobd.spec:516]: what will happen if we delete dattobd installed this way? Will symbol links be deleted too?

File: dist/dattobd.spec (hunk 2 above)
Comment (natalia-zelazna) [reply dist/dattobd.spec:11]: /usr/lib/systemd is on all redhats no wmatter what version and on Ubuntu 20.  
But on Ubuntu 18 there is only /lib/systemd. 
I guess that for backward compatibility reasons Ubuntu 20 has both paths and the same contents in both of them.
I'll try to rephrase the comment.

File: dist/dattobd.spec (hunk 3 above)
Comment (natalia-zelazna) [reply dist/dattobd.spec:516]: No, the original file and the link remain in the filesystem.

Comment (Swistusmen) [reply dist/dattobd.spec:516]: I guess we should do something about it, we already have issues cause by not completly cleaning after dattobd when it is deleted

**Review by Swistusmen — APPROVED** [review]

**Pull request merged**